### PR TITLE
docs: update how-to-write-contracts

### PR DESCRIPTION
**Pull request by classicalliu**
Update to latest CKB changes.

### CODE REVIEW

File: docs/how-to-write-contracts.md [hunk 1]
```diff
@@ -21,43 +21,37 @@ In addition to the different use cases, lock script and type script are also exe
...
 
-Each script has a `type hash` which uniquely identifies the script, for example, the `type hash` of unlock script, is exactly the corresponding `lock` field value in the referenced cell. When calculating type hash for a script, `version`, `binary`, `reference` and `signed_args` will all be used. So another way of looking at `signed_args`, is that it really is a part of the script.
+Each script has a `type hash` which uniquely identifies the script, for example, the `type hash` of unlock script, is exactly the corresponding `lock` field value in the referenced cell. When calculating type hash for a script, `code_hash` and `args` will all be used. So another way of looking at `args`, is that it really is a part of the script.
```
Comment (Mine77) [inline docs/how-to-write-contracts.md]: Do we still have `type hash`?

Comment (xxuejie) [reply docs/how-to-write-contracts.md]: I think we can delete this section about `type hash`

**Review by xxuejie — APPROVED** [review]

**Comment by xxuejie** [comment]
Just one minor comment, then we can merge this.

File: docs/how-to-write-contracts.md (hunk 1 above)
Comment (classicalliu) [reply docs/how-to-write-contracts.md]: deleted

**Pull request merged**